hooks/useUpdateTokenMetadata.ts

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react'
+import axios, { AxiosError } from 'axios'
+import { useGetPoint } from '@/hooks/quest/useGetPoint'
+
+export interface TokenAttribute {
+  display_type: string | undefined
+  trait_type: string
+  value: number | string
+}
+export interface KamonToken {
+  name: string
+  description: string
+  image: string
+  attributes: TokenAttribute[]
+}
+
+export const useUpdateTokenMetadata = () => {  
+  const { refetchPoint } = useGetPoint()
+  const [updateTokenMetadataIsSubmitting, setUpdateTokenMetadataIsSubmitting] = useState<boolean>()
+  const ipfsApiEndpoint = process.env.NEXT_PUBLIC_IPFS_API_URI + ''
+
+  const updateTokenMetadata = async (tokenJSON: KamonToken, userAddress: string) => {
+    setUpdateTokenMetadataIsSubmitting(true)
+    let dateFromToken = 0
+    let rolesFromToken: string[] = []
+    const updatedPointsQuery = await refetchPoint()
```

Comment:
actually fetch is ok, and better to make Type of response for metadata like this one I wrote
https://github.com/henkaku-center/omise-interface/pull/90/files#diff-fc9c9c36fa2146079c5dacf193135b4bf265fa74488ce27d3ad5030156010eddR10

so that, another people will know what kinda of data they need to expect

Reply:
Sorry @SamuraiT, do you mean ditching axios for fetch on this PR?

I did understand the part about the interface, thanks!

Reply:
add Type for response that's what i mean

Reply:
Done in https://github.com/henkaku-center/omise-interface/pull/80/commits/dfc2d7549230802d3d3259f8bd184c53519ad174 and https://github.com/henkaku-center/omise-interface/pull/80/commits/f76e4dd4ee249241bd6da616810c586779d71bbf